transformers/integration_tests/cat_file_test.go

```diff
@@ -68,9 +68,9 @@ var _ = Describe("Cat File transformer", func() {
 			header)
 		Expect(err).NotTo(HaveOccurred())
 
-		headerSyncLogs := test_data.CreateLogs(header.Id, logs, db)
+		EventLogs := test_data.CreateLogs(header.Id, logs, db)
```

Comment:
Do we want lowercase `eventLogs` here? (also in the rest of the integration tests)

Reply:
yes, nice catch!